keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/BuildMetadata.kt

```diff
@@ -13,7 +17,23 @@ data class BuildMetadata(
   val startedTs: String? = null,
```

Comment:
What is the difference between `startedAt` and `startedTs`? Why do we need both?

Reply:
No idea, this is what comes from the API.

Reply:
Yeah, but if we only ever use one of them, do we actually need the other on our model?

Reply:
Fair point, but an unrelated change. I'll defer to @gal-yardeni on what we use

Reply:
We get this info from rocket. I'll take a pass later this week to see if it's still needed, thanks for pointing it out!